cypress/e2e/users/usertype-fields-displays-on-create-user-page.cy.js

```diff
@@ -0,0 +1,87 @@
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import getRandomPostfix, {
+  getTestEntityValue,
+  randomFourDigitNumber,
+} from '../../support/utils/stringTools';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import PatronGroups from '../../support/fragments/settings/users/patronGroups';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import UsersSearchPane from '../../support/fragments/users/usersSearchPane';
+import usersSearchResultsPane from '../../support/fragments/users/usersSearchResultsPane';
+
+describe('Users', () => {
+  let userData;
+  let servicePointId;
+  let newUserId;
+  const patronGroup = {
+    name: getTestEntityValue('groupUserChange'),
+  };
+  const barcodeNumber = getRandomPostfix();
+  const userOne = {
+    patronGroup: patronGroup.name,
+    barcode: barcodeNumber,
+    username: `AutotestUser_${barcodeNumber}`,
+    personal: {
+      lastName: 'User',
+      firstName: 'Delete',
+      email: `tom${randomFourDigitNumber()}@yopmail.com`,
+    },
+  };
+  before('Preconditions', () => {
+    cy.getAdminToken().then(() => {
+      ServicePoints.getViaApi({ limit: 1, query: 'name=="Circ Desk 1"' }).then((servicePoints) => {
+        servicePointId = servicePoints[0].id;
+      });
+      PatronGroups.createViaApi(patronGroup.name).then((patronGroupResponse) => {
+        patronGroup.id = patronGroupResponse;
+      });
+      cy.createTempUser(
+        [
+          Permissions.uiUsersPermissions.gui,
+          Permissions.uiUsersPermissionsView.gui,
+          Permissions.uiUsersCreate.gui,
+          Permissions.uiUsersCreateResetPassword.gui,
+        ],
+        patronGroup.name,
+      ).then((userProperties) => {
+        userData = userProperties;
+        UserEdit.addServicePointViaApi(servicePointId, userData.userId, servicePointId);
+      });
+    });
+  });
+
+  before('Login', () => {
```

Comment:
Please combine 'before' to one

Reply:
combine 'before' into one
then((userProperties) => {
        userData = userProperties;
        UserEdit.addServicePointViaApi(servicePointId, userData.userId, servicePointId);
        cy.login(userData.username, userData.password, {
          path: TopMenu.usersPath,
          waiter: UsersSearchPane.waitLoading,
        });